fortran/src/CMakeLists.txt

```diff
@@ -13,8 +13,10 @@ if (WIN32)
       if (NOT H5_FORTRAN_C_LONG_DOUBLE_IS_UNIQUE)
         set (H5_DBL_NOT_UNIQUE_EXP ";")
       endif ()
+      set (CMAKE_NO_DEPRECATED_SYMBOLS 0)
       if (NOT H5_NO_DEPRECATED_SYMBOLS)
-        set (H5_NO_DEPRECATED_SYMBOLS ";")
+        set (CMAKE_NO_DEPRECATED_SYMBOLS 1)
```

Comment:
This seems to flip whether or not deprecated symbols are enabled - is this intentional?

Reply:
No that was not the intention, it was to only create a variable that could be used in the Fortran include file to create the correct define.

Reply:
See lines 93+ in H5config_f.inc.cmake